riva/proto/riva_tts.proto

```diff
@@ -80,6 +80,9 @@ message SynthesizeSpeechRequest {
   // Zero Shot model params
   ZeroShotData zero_shot_data = 6;
 
+  // user_dictionary parameters
+  string user_dictionary = 7;
```

Comment:
Need to choose a better name here and also add a description for this field, how about `custom_phone_dictionary` as we mention `phone_dictionary_file` argument for users for the default build time IPA dictionary https://docs.nvidia.com/deeplearning/riva/user-guide/docs/tts/tts-custom.html#pretrained-quick-start-pipelines

@mohnishparmar @virajkarandikar as well for review and suggestions.

Reply:
latest proto field name looks fine, resolving